Lock/PolicyView.swift

```diff
@@ -114,19 +118,19 @@ class RuleView: UIView {
             }
         }
 
-        var color: UIColor {
+        func color(style: Style) -> UIColor {
             switch self {
             case .ok:
-                return UIColor(red: 0.502, green: 0.820, blue: 0.208, alpha: 1)
+                return style.primaryColor
             case .error:
-                return UIColor(red: 0.745, green: 0.271, blue: 0.153, alpha: 1)
+                return style.inputBorderColorError
             case .none:
-                return UIColor(red: 0.016, green: 0.016, blue: 0.016, alpha: 1)
+                return style.textColor
             }
         }
     }
 
-    fileprivate func render(text: String, withStatus status: Status) {
+    fileprivate func render(text: String, withStatus status: Status, style: Style) {
         let font = UIFont.systemFont(ofSize: 13)
 
         let attachment = NSTextAttachment()
```

Comment:
How can I tint the icon to be the same color as the text? The green / red that Lock is using for the icons do not match our color scheme

Reply:
You can add a foreground color attribute to the place in the attributed string where the attachment is located. I did something similar to fix the chevron color inhttps://github.com/auth0/Lock.swift/pull/579